cmd/plugin/cli/common.go

```diff
@@ -198,3 +200,55 @@ func parseArgs(args []string) []string {
 
 	return parsedArgs
 }
+
+func installFlagsFilter(config *apiv1.KubectlStorageOSConfig) map[string]string {
+	requiredFlags := make(map[string]string)
+	if config.Spec.Install.EnablePortalManager {
+		requiredFlags[installer.PortalClientIDFlag] = config.Spec.Install.PortalClientID
+		requiredFlags[installer.PortalSecretFlag] = config.Spec.Install.PortalSecret
+		requiredFlags[installer.PortalTenantIDFlag] = config.Spec.Install.PortalTenantID
+		requiredFlags[installer.PortalAPIURLFlag] = config.Spec.Install.PortalAPIURL
+	}
+
+	if !config.Spec.AirGap {
+		return requiredFlags
+	}
```

Comment:
What's the reasoning for this if statement? If we're not airgapped we don't want to check etcd and co required flags?